matplotcheck/base.py

```diff
@@ -1062,62 +1062,89 @@ def assert_line(
         self,
         slope_exp,
         intercept_exp,
-        xtime=False,
+        check_coverage=True,
         message_no_line="Expected line not displayed",
         message_data="Line does not cover data set",
     ):
         """Asserts that there exists a line on Axes `ax` with slope `slope_exp`
-        and y-intercept `intercept_exp` and goes at least from x coordinate
-        `min_val` to x coordinate `max_val`
+        and y-intercept `intercept_exp` and
 
         Parameters
         ----------
         slope_exp : float
             Expected slope of line
         intercept_exp : float
             Expeted y intercept of line
-        xtime : boolean
-            Set ``True`` if x-axis values are datetime
+        check_coverage : boolean
+            If `check_coverage` is `True`, function will check that the goes at
+            least from x coordinate `min_val` to x coordinate `max_val`. If the
+            line does not cover the entire dataset, and `AssertionError` with
+            be thrown with message `message_data`.
```

Comment:
@lwasser so here this new parameter was added in order to avoid coverage checking. After playing with the tests, I noticed this is due to the checks for one-to-one lines. Since one-to-one lines without points on the plot don't necessarily have a minimum x and y value for the points on the plot (since there are none), the test will fail. We could look into other ways of testing this maybe, making it automatically not check coverage for one-to-one lines? Not sure